packages/react/src/components/Accordion/__tests__/__snapshots__/Accordion.Skeleton-test.js.snap

```diff
@@ -51,14 +51,12 @@ exports[`AccordionSkeleton should render 1`] = `
           paragraph={false}
           width="100%"
         >
-          <p
-            className="cds--skeleton__text cds--accordion__title"
-            style={
-              Object {
-                "width": "100%",
-              }
-            }
-          />
+          <div>
+            <p
+              className="cds--skeleton__text cds--accordion__title"
```

Comment:
AccordionItemSkeleton doesn't look correct in the deploy preview
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/3360588/206750518-7a9f471c-4b1d-4619-94fc-93cd09863c73.png">

Reply:
Percy didn't catch this because this story is not included in the e2e tests. Only the default story is:

https://github.com/carbon-design-system/carbon/blob/005e7a595afd7edccbffe5d9fce50a370baa6615/e2e/components/accordion/accordion-test.e2e.js#L15-L25

We might want to double check every component that uses SkeletonText

Reply:
I’ll take a look, I updated a lot of logic in that component so I must have missed something.

Reply:
fixed, there was an extra div being rendered that didn't need to be that was throwing off styles. Should be rendering exactly as it was before now.